src/ext-test/java/org/opentripplanner/ext/vectortiles/layers/stops/StopsLayerTest.java

```diff
@@ -18,9 +31,10 @@
 public class StopsLayerTest {
 
   private RegularStop stop;
+  private RegularStop stop2;
 
   @BeforeEach
-  public void setUp() {
+  public final void setUp() {
```

Comment:
It's not really necessary to mark methods as final especially if you  don't plan to extend the class.